package/network/config/ltq-vdsl-app/files/10_atm_ptm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
> So why shouldn't it also work here?

Mhh, never said that. 

>The ltq-adsl-app uses the dsl_cpe_control-3.24.4.4 package, which also sets the DSL_TC_LAYER_STATUS environment variable for the notify script. So why shouldn't it also work here?

Thanks a lot for the information!

> Well, I have my own DSLAM here, so I have an ADSL line and I have the documentation, but I don't have any hardware.

Bummer! Seams to me a loose/loose situation. But at least I was able to get why ltq-atm-vr9/ltq-ptm-vr9 isn't autoloaded. The vr9 variants have a `AUTOLOAD:=` in the Makefile, which I missed earlier.